src/cftlib/java/uk/gov/hmcts/reform/em/hrs/cftlib/CftLibConfig.java

```diff
@@ -39,7 +39,7 @@ public void configure(CFTLib lib) throws Exception {
         lib.configureRoleAssignments(json);
 
         lib.importDefinition(Files.readAllBytes(
-            Path.of("src/functionalTest/resources/CCD_HRS_v1.1-AAT.xlsx")));
+            Path.of("src/functionalTest/resources/CCD_HRS_v1.2-AAT.xlsx")));
```

Comment:
unrelated changes like this one and other renaming stuff can go separately, so if we break something we will revert just the related change and reviewing would easy.